src/Expanders.jl

```diff
@@ -273,6 +273,7 @@ end
 # -----
 
 function Selectors.runner(::Type{DocsBlocks}, x, page, doc)
+    id_str = match(r"^@docs[ ]?(.*)$", first(split(x.language, ';', limit = 2)))[1]
```

Comment:
Perhaps the syntax should be
````
```@docs id="something"
```
````
instead? I think that pattern is used elsewhere for e.g. doctest filters etc.

Reply:
Also, do you ever care about the `id` or just that it is unique?

Reply:
Maybe the current approach follows the idea that is used in example blocks? there the id/name for continued examples is also just added with a space.

Reply:
Yea that's true.

Reply:
> Also, do you ever care about the `id` or just that it is unique?

If I remember correctly, I need the `id_str` for the filtering mechanism in `@index` blocks.
This filtering should allow to 
* exclude docstrings with ids that are not empty from the listing 
* and to list only a subset of docstrings.

Reply:
is it ever useful to have index with duplicates though? Perhaps there should be a canonical `@docs` block or something?

Reply:
>is it ever useful to have index with duplicates though?

I guess not, and I am not sure myself anymore what I wanted this feature for.
I think it was primarily meant for *exclusion* of docstring subsets, but most of the use cases that come to mind can also be achieved with the `Pages` kw.
The only (exotic) example I can think of is a single documentation page that has subsections (possibly containing duplicate docstrings in `@docs` blocks) and an `@index` is meant to be created for each subsection.